app/boards/shields/iris/iris.keymap

```diff
@@ -1,6 +1,3 @@
-# Copyright (c) 2020 Pete Johanson, Kurtis Lew
```

Comment:
We should keep the copyright header, just use valid comments for DT. I believe this needs `/*  */` style comments.